CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -55,7 +55,7 @@ class EditDemographicsViewController: UIViewController {
             .font: UIFont.getFont(.medium, size: 24),
         ]
         
-        backButton.setImage(UIImage(named: "back_arrow"), for: .normal)
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
         backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
         backBarButtonItem = UIBarButtonItem(customView: backButton)
```

Comment:
Relates to an above comment, but is there a reason you save the UIBarButtonItem as a variable, or would it be fine to just do `navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)`?